### PR TITLE
Use node to verify, format, and aggregate assets

### PR DESCRIPTION
**Pull request by davepuchyr**
Git hooks automagically generate assets.json when a new asset is added.

### CODE REVIEW

File: assets.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "caip-20": "",
+    "iov-name-service-uri": "asset:atom",
```
Comment (davepuchyr) [inline assets.json:4]: @antoineherzog , do we want to change `iov-name-service-uri` to `starname-uri` or simply `uri`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Digital Assets Directory #
```
Comment (davepuchyr) [inline README.md:1]: @antoineherzog / @BenSim, is more required in the README?  If so then please add it.

Comment (BenSim) [reply README.md:1]: good with me

File: assets.json (hunk 1 above)
Comment (antoineherzog) [reply assets.json:4]: Lets change to starname-uri just to be explicit for developers I think

**Pull request merged**